.github/workflows/github-actions-pr-on-master.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       # Setup nodejs
       - uses: actions/setup-node@v1
         with:
-          node-version: '12.14.x'
+          node-version: '16.14.x'
```

Comment:
Can you update this on package.json -> engines? Currently, it has 16.13.x in it.  Check if we need to update that too.

Reply:
Can you please elaborate this point? Unable to see node version on the path you mentioned.

Reply:
Package.json allows adding a `engines` section where we can enforce which node, and npm versions should be allowed. Add a similar one on the package.json.
```
  "engines": {
        "npm": ">=8.1.2",
        "node": ">=16.13.1"
    },
```

Reply:
As suggested, I haved added engines in package.json file